spec/views/notes/show.html.erb_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe "notes/show", type: :view do
```

Comment:
I consider view tests that check for specific strings on a page kind of useless, as they get stale extremely quickly. I'd vote to simply remove them. What do you think?

Reply:
I'd suggest instead of checking for specific strings, we check for the values of the assigned note.

Reply:
Fixed